src/components/PostInfo/PostInfo.scss

```diff
@@ -1 +1,16 @@
-// add styles here
+.PostInfo {
+  margin: 10px auto;
+  width: 90%;
+  border: 1px solid #000;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)